dotcom-rendering/src/web/components/Card/Card.tsx

```diff
@@ -307,80 +306,79 @@ export const Card = ({
 					imageUrl !== undefined ? imagePositionOnMobile : 'top'
 				}
 				minWidthInPixels={minWidthInPixels}
+				imageType={imageType}
 			>
-				{imageType === 'mainmedia' && (
```

Comment:
A key part of this PR is that avatars and mainmedia images now share the same location (and associated wrappers)

Reply:
This is a really nice solution!